booru/templates/booru/posts/view.html

```diff
@@ -98,11 +97,66 @@ <h3>Options</h3>
 				</div>
 			</div>
 		</div>
+		<div id="comment-section">
+			<span class="title"><a href="#">Comments</a></span>
```

Comment:
Maybe add an arrow pointing down when it is open and up when it is closed, kinda like on the tags page